.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Is this file used by evergreen? If not remove?

Reply:
To test our code works on [this](https://github.com/10gen/buildhost-configuration/blob/master/packer/ubuntu1804-ecs-image.json) Ubuntu 18.04 ECS container (docker) image we set one up from the EC2 machine, copy this file and the src folder over to it and run this script to test that we can authenticate.

Reply:
"run aws ECS auth test" passes in the directory to copy, _project_dir_, to [aws_e2e_ecs.js](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/aws_e2e_ecs.js). That mongo shell script uses [container_tester.py](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/container_tester.py) ([here](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/aws_e2e_ecs.js#L14-L27)) to set up an ECS container and copy over _project_dir_. Once everything is copied over [ecs_hosted_test.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/ecs_hosted_test.sh) is run which runs the mongo shell script [ecs_hosted_test.js](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/ecs_hosted_test.js) and returns whether or not that shell script succeeded or failed. Lastly, [ecs_hosted_test.js](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/ecs_hosted_test.js) has the [program](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/ecs_hosted_test.js#L27) it's going to run and the [uri](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_aws/lib/ecs_hosted_test.js#L26) to give it hard-coded into the script.